tensor_comprehensions/pybinds/pybind_engine.cc

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <Python.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <ATen/ATen.h>
+
+#include "pybind_utils.h"
+#include "tc/aten/aten_compiler.h"
+#include "tc/core/compilation_cache.h"
+#include "tc/core/flags.h"
+#include "tc/core/mapping_options.h"
+#include "tc/core/scope_guard.h"
+
+namespace tc {
+namespace python {
+
+namespace py = pybind11;
+
+PYBIND11_MODULE(tc, m) {
+  m.def(
+      "global_debug_init", // exposing the debugging flags to people
+      [](std::vector<std::string> args) {
+        if (args.size() > 0) {
+          args.insert(args.begin(), "tc");
+        }
+        int numArgs = args.size();
+        // now we construct a char** argv type from args
+        std::vector<char*> vargs; // char* vector args
+        for (auto& arg : args) {
+          vargs.push_back(const_cast<char*>(arg.data()));
+        }
+        char** argv = vargs.data();
+        tc::python::globalDebugGflagsGlogInit(&numArgs, &argv);
```

Comment:
this function returns an unused ```bool```, what is the rationale here?

Reply:
this is how gflags/glog work. if the init fails at any point, we get the exception otherwise, it's important to catch True everywhere for successful init. There is InitGoogleTest and InitGoogleLogging and ParseCommandLineFlags that all should pass. If any of them fails, the init was problematic either due to bad flags, etc. 

For reference, this is also how things are in main in C++ tests.

Reply:
#resolved (please comment if some other clarity is needed). Thanks.